packages/core/package.json

```diff
@@ -14,12 +14,12 @@
   "bugs": "https://github.com/paritytech/polkadot-onboard.git/issues",
   "homepage": "https://github.com/paritytech/polkadot-onboard.git#readme",
   "license": "Apache-2.0",
-  "main": "dist/index.js",
-  "types": "dist/types/index.d.ts",
-  "type": "module",
```

Comment:
This should bot be removed! This lib is intended to be only imported as esm. This is why type:'module' was added in the new version.